modules/virt-accessing-exported-vm-manifests.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-exporting-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-accessing-exported-vm-manifests_{context}"]
+= Accessing exported virtual machine manifests
+
+After you export a virtual machine (VM) or snapshot, you can get the `VirtualMachine` manifest and related information from the export server.
+
+.Prerequisites
+
+* You exported a virtual machine or VM snapshot by creating a `VirtualMachineExport` custom resource (CR).
++
+[NOTE]
+====
+`VirtualMachineExport` objects that have `spec.source.kind: PersistentVolumeClaim` do not generate virtual machine manifests.
+====
+
+.Procedure
+
+. To access the manifests, you must first copy the certificates from the source cluster to the target cluster.
+
+.. Log in to the source cluster.
+
+.. Save the certificates to the file `cacert.crt` by running the following command:
++
+[source,terminal]
+----
+$ oc get vmexport <export_name> -o jsonpath={.status.links.external.cert} > cacert.crt <1>
+----
+<1> Replace `<export_name>` with the `metadata.name` value from the `VirtualMachineExport` object.
+
+.. Copy the `cacert.crt` file to the target cluster.
+
+. Decode the token in the source cluster and save it to the file `token_decode` by running the following command:
++
+[source,terminal]
+----
+$ oc get secret export-token-<export_name> -o jsonpath={.data.token} | base64 --decode > token_decode <1>
+----
+<1> Replace `<export_name>` with the `metadata.name` value from the `VirtualMachineExport` object.
+
+. Copy the `token_decode` file to the target cluster.
+
+. Get the `VirtualMachineExport` custom resource by running the following command:
++
+[source,terminal]
+----
+$ oc get vmexport <export_name> -o yaml
+----
+
+. Review the `status.links` stanza, which is divided into `external` and `internal` sections. Note the `manifests.url` fields within each section:
```

Comment:
```suggestion
. Review the `status.links` path, which is divided into `external` and `internal` sections. Note the `manifests.url` fields within each section:
```

Reply:
I'm used to using "stanza" as described here:  https://en.wikipedia.org/wiki/Stanza_(computing) 
Though I'm not finding a lot of YAML-specific references to "stanza," there is [this documentation](https://www.playframework.com/documentation/1.2/yaml) that uses it like so: "Each stanza must start with the name of the model class..."